Orange/classification/softmax_regression.py

```diff
@@ -10,26 +11,40 @@ class SoftmaxRegressionLearner(Learner):
     """L2 regularized softmax regression classifier.
     Uses the L-BFGS algorithm to minimize the categorical
     cross entropy cost with L2 regularization. This model is suitable
-    when dealing with a multi-class classification problem
+    when dealing with a multi-class classification problem.
 
     When using this learner you should:
 
     - choose a suitable regularization parameter lambda\_,
-    - continuize all discrete attributes,
-    - consider appending a column of ones to the data set (intercept term),
-    - transform the data set so that the columns are on a similar scale,
     - consider using many logistic regression models (one for each
       value of the class variable) instead of softmax regression.
 
-    lambda\_ : float, optional (default=1.0)
-       the regularization parameter. Higher values of lambda\_
-       force the coefficients to be smaller
+    Parameters
+    ----------
+
+    lambda_ : float, optional (default=1.0)
```

Comment:
Underscores should be escaped ("lambda\_") otherwise sphinx has problems while compiling the documentation.